next.config.js

```diff
@@ -45,198 +40,59 @@ if (typeof require !== 'undefined') {
   require.extensions['.less'] = () => { };
 }
 
-const srcFolder = [
-  path.resolve(__dirname, './src/components'),
-  path.resolve(__dirname, './src/utils'),
-  path.resolve(__dirname, './src/pages'),
-  path.resolve(__dirname, './src/redux'),
-];
-
-const generalWebpackConfig = [
-  {
-    test: /\.(ttf|eot|svg)$/,
-    use: {
-      loader: 'file-loader',
-      options: {
-        name: 'fonts/[hash].[ext]',
+const nextConfig = {
+  // Redirects enforced by Next.
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/data-management',
+        permanent: false,
       },
-    },
-  },
-  {
-    test: /\.(woff|woff2)$/,
-    use: {
-      loader: 'url-loader',
-      options: {
-        name: 'fonts/[hash].[ext]',
-        limit: 5000,
-        mimetype: 'application/font-woff',
+      {
+        source: '/experiments',
+        destination: '/data-management',
+        permanent: false,
```

Comment:
why do we need this one as well?

Reply:
I guess in order to make the URL scheme more consistent. Right now when you click on a link to an experiment, it goes `/experiments/[experimentId]/data-exploration` for example. A user may try to access the URL under `/experiments`, assuming this is where they'd find their experiments. A redirect to data management makes sense for this. This was here before as you will see in the diff.

Reply:
Yes, I remember it was here, I was just wondering whether it is still needed. Sounds like it still is, let's leave it.